observability-lib/grafana/variables.go

```diff
@@ -12,30 +12,44 @@ type VariableOption struct {
 
 type QueryVariableOptions struct {
 	*VariableOption
-	Datasource string
-	Query      string
-	Multi      bool
-	Regex      string
+	Datasource   string
+	Query        string
+	Multi        bool
+	Regex        string
+	CurrentText  string
+	CurrentValue string
+	IncludeAll   bool
 }
 
 func NewQueryVariable(options *QueryVariableOptions) *dashboard.QueryVariableBuilder {
+	if options.CurrentText == "" {
```

Comment:
I think I suggested this before when it came to legends, but can't you do default values in the struct itself?